lib/__tests__/canary.test.ts

```diff
@@ -66,45 +68,13 @@ test('correctly creates canary', () => {
           Name: 'SCRIPT_S3_BUCKET',
           Type: 'PLAINTEXT',
           Value: {
-            Ref: 'AssetParameters3d34b07ba871989d030649c646b3096ba7c78ca531897bcdb0670774d2f9d3e4S3BucketDA91EFBC',
+            'Fn::Sub': 'cdk-hnb659fds-assets-${AWS::AccountId}-${AWS::Region}',
           },
         },
         {
           Name: 'SCRIPT_S3_KEY',
           Type: 'PLAINTEXT',
-          Value: {
-            'Fn::Join': [
-              '',
-              [
-                {
-                  'Fn::Select': [
-                    0,
-                    {
-                      'Fn::Split': [
-                        '||',
-                        {
-                          Ref: 'AssetParameters3d34b07ba871989d030649c646b3096ba7c78ca531897bcdb0670774d2f9d3e4S3VersionKeyF3F83F76',
-                        },
-                      ],
-                    },
-                  ],
-                },
-                {
-                  'Fn::Select': [
-                    1,
-                    {
-                      'Fn::Split': [
-                        '||',
-                        {
-                          Ref: 'AssetParameters3d34b07ba871989d030649c646b3096ba7c78ca531897bcdb0670774d2f9d3e4S3VersionKeyF3F83F76',
-                        },
-                      ],
-                    },
-                  ],
-                },
-              ],
-            ],
-          },
+          Value: '3d34b07ba871989d030649c646b3096ba7c78ca531897bcdb0670774d2f9d3e4.zip',
```

Comment:
This is probably going to happen in `aws-cdk` when we start running tests for v2.